common/src/main/java/tachyon/util/network/NetworkAddressUtils.java

```diff
@@ -84,10 +164,14 @@ public static String getLocalHostName(int timeout) {
   }
 
   /**
+   * Check if the underlying OS is Windows.
+   */
+  public static final boolean WINDOWS = System.getProperty("os.name").startsWith("Windows");
```

Comment:
We discussed this comment two weeks ago. It was using "SystemUtils.IS_OS_WINDOWS". 
   
calvinjia added a note 19 days ago
I think we use System.getProperty("os.name").startsWith("Windows"); in other places, if possible let's try to be consistent. If this way is better, let's open a ticket to use this throughout the codebase